packages/locales/lib/human/de.json

```diff
@@ -763,5 +763,22 @@
   "enter_translation": "Übersetzung eingeben",
   "individual_filters": "individuell gefiltert",
   "best_buddy": "Bester Kumpel",
-  "data_limit_reached": "Du hast kürzlich zu viele Daten angefordert und befindest dich im Cooldown bis {{until}}"
+  "data_limit_reached": "Du hast kürzlich zu viele Daten angefordert und befindest dich im Cooldown bis {{until}}",
+  "unknown_station": "unbekannte Kraftquelle",
+  "exclude_battle": "Dyna-Raids ausschließen",
+  "station": "Kraftquelle",
+  "stations": "Dynamax",
+  "stations_options": "Kraftquellen Optionen",
+  "all_stations": "alle Kraftquellen",
+  "search_battles": "Suche Dyna-Raids",
+  "started": "gestartet",
+  "ended": "beendet",
+  "global_search_stations": "Füge den Namen der Kraftquelle ein...",
+  "station_timers": "Kraftquellen Timer",
+  "station_opacity": "Kraftquellen Durchsichtigkeit",
+  "max_battles": "Dyna-Raids",
+  "dynamax": "Dynamax",
+  "stations_subtitle": "Kraftquellen auf der Map anzeigen",
+  "dynamax_subtitle": "Dyna-Raids auf der Map anzeigen",
+  "override": "Dyna-Raid Override"
```

Comment:
is this "override" just used for Dyna-Raid or for every other preset-filter like raids?

Reply:
@TurtIeSocks did you publish this key on purpose like that? instead `dyna_override` like for `raid_override` ? :)